bundle/edu.gemini.spdb.rollover.servlet/src/main/scala/edu/gemini/rollover/servlet/RolloverServlet.scala

```diff
@@ -55,10 +55,12 @@ final class RolloverServlet(site: Site, odb: IDBDatabaseService, user: java.util
     <obs>
       <id>{ ro.id.toString }</id>
       <partner>{ ro.partner }</partner>
-      <target>
-        <ra>{ ro.target.coords.map(_.ra).getOrElse(0.0) }</ra>
-        <dec>{ ro.target.coords.map(_.dec).getOrElse(0.0) }</dec>
-      </target>
+      {ro.targets.map { t =>
+        <target>
+          <ra>{  t.coords.map(_.ra).getOrElse(0.0) }</ra>
+          <dec>{ t.coords.map(_.dec).getOrElse(0.0) }</dec>
+        </target>
+      }}
```

Comment:
Rollover servlet is used by ITAC. We should be careful here to avoid changes that impact the output especially when we are 2 weeks away from the next ITAC meeting